sdk/identity/Azure.Identity/src/Credentials/ManagedIdentityCredential.cs

```diff
@@ -33,11 +33,11 @@ protected ManagedIdentityCredential()
         { }
 
         /// <summary>
-        /// Creates an instance of the ManagedIdentityCredential capable of authenticating a resource with a managed identity.
+        /// Creates an instance of <see cref="ManagedIdentityCredential"/> capable of authenticating a resource with a user-assigned or a system-assigned managed identity.
         /// </summary>
         /// <param name="clientId">
-        /// The client ID to authenticate for a user-assigned managed identity. More information on user-assigned managed identities can be found at
-        /// <see href="https://learn.microsoft.com/entra/identity/managed-identities-azure-resources/overview#how-a-user-assigned-managed-identity-works-with-an-azure-vm"/>.
+        /// The client ID to authenticate for a <see href="https://learn.microsoft.com/entra/identity/managed-identities-azure-resources/overview#how-a-user-assigned-managed-identity-works-with-an-azure-vm">user-assigned managed identity</see>.
+        /// If not provided, a system-assigned managed identity is used.
```

Comment:
Did you mean to say user-assigned here?

```suggestion
        /// If not provided, a user-assigned managed identity is used.
```

Reply:
My understanding is that system-assigned is used when no client ID is passed to the ctor. Is that correct? Here's what I was looking at: https://github.com/Azure/azure-sdk-for-net/blob/d949e9dddefb04104aa659080b59fa577262ee40/sdk/identity/Azure.Identity/src/ManagedIdentityClient.cs#L49